code/modules/surgery/surgery_step.dm

```diff
@@ -78,7 +78,7 @@
 	else if(locate(/obj/structure/bed, T))
 		propability = 0.5
 
-	return propability + sleepbonus - selfpenalty
+	return max(propability + sleepbonus - selfpenalty, 0.1)
```

Comment:
propability

Reply:
always been that way and i didn't bother to spell it right when porting speed
it kinda feels intentional based on how astray it is from the original word

Reply:
its not very astray, its 1 letter different

Reply:
oh nevermind its not meant to be probability, looks like they just made a word or something

Reply:
> 
> 
> its not very astray, its 1 letter different

well you don't pronounce the b anything like a p

Reply:
its not meant to be probability, I think its propability because its how effective the props are????